.yamllint.yml

```diff
@@ -0,0 +1,8 @@
+---
+extends: default
+rules:
+  line-length: disable
+  key-duplicates:
+    ignore: |
+      templates/http/base/deployment-model-server.yaml
+      templates/http/base/model-config.yaml
```

Comment:
any reason why we are explicitly ignoring this?

Reply:
there are multiple definitions of the same key in these files, with conditional statements that the linter can't interpret.

like so:
```yaml
data:
  {%- if values.existingModelServer %}
  MODEL_ENDPOINT: "{{values.modelEndpoint}}"
  {%- else %}
  MODEL_ENDPOINT: "http://{{values.name}}-model-server:{{values.modelServicePort}}"
  {%- endif %}
```

it should only ignore the duplicate keys error though